TTHAnalysis/cfg/run_susyDeDx_cfg.py

```diff
@@ -90,6 +92,8 @@
 autoAAA(mcSamples)
 for c in mcSamples:
     c.triggers = triggers_1mu_iso
+    c.triggers += triggers_1e_iso + triggers_1e_noniso
+    c.triggers += triggers_SOS_highMET
```

Comment:
bug here: after line 94, c.triggers is equal to triggers_1mu_iso (by reference) and so the two remaining lines add the new triggers also to the triggers_1mu_iso list.
so, the fix can be either to change the first line to `c.triggers = triggers_1mu_iso[:]` (to force to make copy of the list by value) or `c.triggers = triggers_1mu_iso + triggers_1e_iso + triggers_1e_noniso + triggers_SOS_highMET`